app/code/core/Mage/Core/Model/App.php

```diff
@@ -384,15 +384,10 @@ public function run($params)
      */
     protected function _initEnvironment()
     {
-        if (Mage::getIsDeveloperMode() && class_exists('\Spatie\Ignition\Ignition')) {
-            \Spatie\Ignition\Ignition::make()
-                ->applicationPath(Mage::getBaseDir())
-                ->register();
-        } else {
-            $this->setErrorHandler(self::DEFAULT_ERROR_HANDLER);
-        }
-
+        Mage::dispatchEvent('core_app_init_before', ['app' => $this]);
+        $this->setErrorHandler(self::DEFAULT_ERROR_HANDLER);
         date_default_timezone_set(Mage_Core_Model_Locale::DEFAULT_TIMEZONE);
+
```

Comment:
```suggestion
Mage::dispatchEvent('core_app_init_environment_after', ['app' => $this]);

```

Reply:
at the moment I'd only add this event to this class, it allows you to override what the method does and it should be enough at the moment